contrib/go-chi/chi.v5/appsec.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package chi
+
+import (
+	"net/http"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/httpsec"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func withAppsec(next http.Handler, r *http.Request, span tracer.Span) http.Handler {
+	var pathParams map[string]string
+	if routeCtx := chi.RouteContext(r.Context()); routeCtx != nil {
```

Comment:
Can you please apply the same comment here and elsewhere please? Also, it might be more tidy to call it `rctx`, but up to you. The last bit is just a nit.

Reply:
Done in https://github.com/DataDog/dd-trace-go/pull/1130/commits/2ed47cc50e51f71fd5929f3ca8eca56d850eedc2 and https://github.com/DataDog/dd-trace-go/pull/1130/commits/2ed47cc50e51f71fd5929f3ca8eca56d850eedc2